### PR TITLE
docs: CWC-288 Revise README.md

### PR DESCRIPTION
**Pull request by iancoolbitx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Depending on your platform, you may can choose different [transport](#Transport)
 
 ### 2. Register and setup hardware wallet.
 
-To register you application with the wallet, take a look at the [wallet module](/packages/cws-wallet). This giud you through the process of registeration and seed generation.
+To register you application with the wallet, take a look at the [wallet module](/packages/core/src/apdu/wallet)  in `core` package. This giud you through the process of registeration and seed generation.
```
Comment (skchang0710) [inline README.md:25]: giud ? 應該是錯字嗎 ？是不是 guide

Comment (iancoolbitx) [reply README.md:25]: ok

**Review by hankliu418 — APPROVED** [review]

**Review by skchang0710 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (delbinwen) [reply README.md]: This guides you through the process of registration and seed generation.

File: README.md [hunk 2]
```diff
@@ -41,45 +41,174 @@ To communicate with CoolWallet device, you need to specify a bluetooth transport
...
+| [`@coolwallet/atom`](/packages/cws-atom) | ![version](https://img.shields.io/npm/v/@coolwallet/atom) | Cosmos |
+| [`@coolwallet/bch`](/packages/cws-bch) | ![version](https://img.shields.io/npm/v/@coolwallet/bch) | Bitcoin Cash |
+| [`@coolwallet/bnb`](/packages/cws-bnb) | ![version](https://img.shields.io/npm/v/@coolwallet/bnb) | Binance |
+| [`@coolwallet/bsc`](/packages/cws-bsc) | ![version](https://img.shields.io/npm/v/@coolwallet/bsc) | Binance Smart Chain |
+| [`@coolwallet/btc`](/packages/cws-btc) | ![version](https://img.shields.io/npm/v/@coolwallet/btc) | Bitcoin/USDT |
+| [`@coolwallet/dot`](/packages/cws-dot) | ![version](https://img.shields.io/npm/v/@coolwallet/dot) | Polkadot/Kusama |
+| [`@coolwallet/eth`](/packages/cws-eth) | ![version](https://img.shields.io/npm/v/@coolwallet/eth) | Ethereum (Ether, ERC20, Smart Contract, EIP-1559 etc.)|
+| [`@coolwallet/icx`](/packages/cws-icx) | ![version](https://img.shields.io/npm/v/@coolwallet/icx) | Icon |
+| [`@coolwallet/ltc`](/packages/cws-ltc) | ![version](https://img.shields.io/npm/v/@coolwallet/ltc) | LetCoin |
```
Comment (delbinwen) [inline README.md:62]:
```suggestion
| [`@coolwallet/ltc`](/packages/cws-ltc) | ![version](https://img.shields.io/npm/v/@coolwallet/ltc) | Litecoin |
```

File: README.md [hunk 3]
```diff
@@ -41,45 +41,174 @@ To communicate with CoolWallet device, you need to specify a bluetooth transport
...
+| [`@coolwallet/trx`](/packages/cws-trx) | ![version](https://img.shields.io/npm/v/@coolwallet/trx) | Tron |
+| [`@coolwallet/xlm`](/packages/cws-xlm) | ![version](https://img.shields.io/npm/v/@coolwallet/xlm) | Stellar/Kinesis |
+| [`@coolwallet/xrp`](/packages/cws-xrp) | ![version](https://img.shields.io/npm/v/@coolwallet/xrp) | Ripple |
+| [`@coolwallet/zen`](/packages/cws-zen) | ![version](https://img.shields.io/npm/v/@coolwallet/zen) | Zen Cash |
 
-Other supported coins: BTC, BCH, ZEN. Open an issue if you want the sdk of any one of them to come out first.
 
-### Other packages
+## Examples: Build ETH in web app
+### Create connect
```
Comment (delbinwen) [inline README.md:70]:
```suggestion
### To connect to CoolWallet Pro via BLE
```

File: README.md [hunk 4]
```diff
@@ -41,45 +41,174 @@ To communicate with CoolWallet device, you need to specify a bluetooth transport
...
+
+disconnect = () => {
+  WebBleTransport.disconnect(this.state.transport.device.id);
+  this.setState({ transport: undefined, cardName: "" });
+};
+
+```
+
+- transport: The object use to communicate with CoolWallet
+- SEPublicKey: The key use to check se.
```
Comment (delbinwen) [inline README.md:106]:
```suggestion
- SEPublicKey: The key used to authenticate SE.
```

File: README.md [hunk 5]
```diff
@@ -41,45 +41,174 @@ To communicate with CoolWallet device, you need to specify a bluetooth transport
...
+
+Obtain app key pairs.
+
+```javascript
+const keyPair = crypto.key.generateKeyPair()
+localStorage.setItem('appPublicKey', keyPair.publicKey)
+localStorage.setItem('appPrivateKey', keyPair.privateKey)
+```
+
+- keyPair: The keys use to check your app.
```
Comment (delbinwen) [inline README.md:116]: shall we move this part to next section?

File: README.md [hunk 6]
```diff
@@ -41,45 +41,174 @@ To communicate with CoolWallet device, you need to specify a bluetooth transport
...
+
+### Register card
```
Comment (delbinwen) [inline README.md:118]:
```suggestion
### Register application with CoolWallet Pro
```

File: README.md [hunk 7]
```diff
@@ -41,45 +41,174 @@ To communicate with CoolWallet device, you need to specify a bluetooth transport
...
+
+Register card and obtain the appId.
+
+```javascript
+const name = 'your app name'
+const SEPublicKey = localStorage.getItem('SEPublicKey')
+const appId = await apdu.pair.register(transport, appPublicKey, password, name, SEPublicKey);
```
Comment (delbinwen) [inline README.md:125]: shall we explain what `password` represent?

Comment (delbinwen) [reply README.md:125]: we could also leave note here to indicate a single CoolWallet Pro could only be paired to 3 devices.

File: README.md [hunk 8]
```diff
@@ -41,45 +41,174 @@ To communicate with CoolWallet device, you need to specify a bluetooth transport
...
+```
+
+If you want to create seed by card, you can use function `createSeedByCard`. And also choose the length of seed(12, 18, 24).
+
+```javascript
+await apdu.wallet.createSeedByCard(transport, appId, appPrivateKey, 12);
+```
+
+
+### Integration
```
Comment (delbinwen) [inline README.md:145]:
```suggestion
### Use coin app
```

File: README.md [hunk 9]
```diff
@@ -41,45 +41,174 @@ To communicate with CoolWallet device, you need to specify a bluetooth transport
...
+};
 
-If you want to build your own app with this sdk, you might find the following repos useful:
+const signedTx = await ETH.signTransaction(signTxData);
 
-- [React Example](https://github.com/antoncoding/cws-web-ble-demo) (with `web-ble`)
+```
 
-- [React-Native Example](https://github.com/kunmingLiu/cws-rn-ble-demo) (with `rn-ble`)
+## Integration
```
Comment (delbinwen) [inline README.md:212]:
```suggestion
## Contributing
```

File: README.md [hunk 10]
```diff
@@ -41,45 +41,174 @@ To communicate with CoolWallet device, you need to specify a bluetooth transport
...
 
-If you build something new, welcome to contact us to put your work in the list.
+If you want to integrate the other coin into CoolWallet, you can refer to [CONTRIBUTING](./CONTRIBUTING.md) or contact us.
```
Comment (delbinwen) [inline README.md:214]: If you're interested to develop new coin for CoolWallet Pro, please see [CONTRIBUTING](./CONTRIBUTING.md) for more information.

File: README.md (hunk 2 above)
Comment (iancoolbitx) [reply README.md:62]: ok

File: README.md (hunk 3 above)
Comment (iancoolbitx) [reply README.md:70]: ok

File: README.md (hunk 4 above)
Comment (iancoolbitx) [reply README.md:106]: ok

File: README.md (hunk 6 above)
Comment (iancoolbitx) [reply README.md:118]: ok

File: README.md (hunk 8 above)
Comment (iancoolbitx) [reply README.md:145]: ok

File: README.md (hunk 9 above)
Comment (iancoolbitx) [reply README.md:212]: ok

File: README.md (hunk 5 above)
Comment (iancoolbitx) [reply README.md:116]: I will move this code to `Register application with CoolWallet Pro`

File: README.md (hunk 7 above)
Comment (iancoolbitx) [reply README.md:125]: ok

**Review by skchang0710 — APPROVED** [review]

File: README.md [hunk 11]
```diff
@@ -125,6 +126,11 @@ const SEPublicKey = localStorage.getItem('SEPublicKey')
 const appId = await apdu.pair.register(transport, appPublicKey, password, name, SEPublicKey);
 ```
 
+- password: Need to set the password, which will be used to change app.
```
Comment (delbinwen) [inline README.md:129]:
```suggestion
- password: Pairing password for the app to establish the connection with CoolWallet Pro. The password could be supplied by the user.
```

Comment (delbinwen) [reply README.md:129]: Does the password have a max length limit?

**Review by skchang0710 — APPROVED** [review]

**Review by hankliu418 — APPROVED** [review]

**Pull request merged**